### ISSUE TITLE
Low accuracy without sports1m model

### ISSUE THREAD

**Issue by jordantrc** [hx173149/C3D-tensorflow#100]
When I train the network from scratch using any of the pre-made training/testing lists from the UCF101 dataset, I get much lower accuracy than advertised on this repository (~25%). I am trying to train from scratch, so I am not loading the Sports1m pre-trained model. I commented-out the tf.transpose operation in the inference_3d function per the documentation.

With the sports1m model, I am able to get about 80% accuracy. If I try to fine-tune from this model, the accuracy actually drops.

I am using a 5 frame per second extract from the UCF 101 videos. I used the shell script included in the repo to produce the frames for the dataset.

Have you seen this issue in the past and do you have any suggestions for improving accuracy?

Thanks!

**Comment by JoseponLee** [comment]
I meet the same problem,how did you slove it

**Comment by MClarkTurner** [comment]
I'm a collaborator with OP. We were unable to solve this issue and were forced instead to train the model in caffe and then extract the layers into a tensorflow format.
Here are the files for doing so: https://github.com/AssistiveRoboticsUNH/IAD-Generator/tree/master/caffe3d/c3d_code/convert_caffe_to_tf
